src/LinearSolvers/LinearSolvers.jl

```diff
@@ -251,9 +251,10 @@ function ldiv!(solver::DQGMRES,
 )
     P = solver.precond.P
     CUDA.allowscalar() do
-        global (y[:], status) = Krylov.dqgmres!(solver.inner, J, x; N=P)
+        Krylov.dqgmres!(solver.inner, J, x; N=P)
```

Comment:
ouch ... why did we have a `global` here?

Reply:
Because of the do loop with its own namespace. I don't think `global` is a problem there or? I thought it only moves the variable one scope up in this case.

Reply:
In my understanding it becomes a global variable available everywhere ^^ 
In that case, I prefer to use the keyword `local`